plugins/woocommerce/client/legacy/js/admin/woocommerce_admin.js

```diff
@@ -361,6 +361,26 @@
 			$( this ).trigger( 'focus' );
 		} );
 
+		$( '#set-post-thumbnail' )
```

Comment:
Since the `#set-post-thumbnail` element is added via a filter, and thus could be altered by another plugin, we should account for that and handle the scenario where this element doesn't exist.